compactcert/builder.go

```diff
@@ -21,12 +21,12 @@ import (
 	"database/sql"
 	"encoding/binary"
 	"fmt"
+	"github.com/algorand/go-algorand/ledger"
```

Comment:
why ? it was good before..

Reply:
goland is moving stuff behind my back, fixed